src/test/java/store/controller/ReviewControllerTest.java

```diff
@@ -0,0 +1,45 @@
+package store.controller;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
+import store.dto.ReviewDTO;
+import store.service.ReviewService;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ReviewControllerTest {
+    @Mock
+    private ReviewService reviewService;
+    @InjectMocks
+    private ReviewController reviewController;
+    private ReviewDTO review = new ReviewDTO(1L, 5, "Test Review");
+
+    @Test
+    public void testGetProducts() {
+        //given
+        List<ReviewDTO> reviews = Collections.singletonList(review);
+        //when
+        Mockito.when(reviewService.getReviews(Mockito.anyLong())).thenReturn(reviews);
```

Comment:
Please avoid `any()` checks in when
Point is in using exact the same params as service send
better use `any() `in verifies